src/index.js

```diff
@@ -3,9 +3,10 @@ var WriteData = require('./write-data');
 
 var main = function(options) {
   console.log("Gathering Manifest data...");
-  var resources = WalkManifest(options.decrypt, options.output, options.input);
-
-  return WriteData(options.decrypt, options.concurrency, resources);
+  var resources = [];
+  WalkManifest(options.decrypt, options.output, options.input, false, 0, resources, function(){
+    return WriteData(options.decrypt, options.concurrency, resources);
```

Comment:
returning here will not return the data from the `main` method. You'd need to add a callback to `main` to accept the results.